sycl/include/sycl/detail/pi.h

```diff
@@ -1398,8 +1398,9 @@ __SYCL_EXPORT pi_result piMemBufferPartition(
 /// Gets the native handle of a PI mem object.
 ///
 /// \param mem is the PI mem to get the native handle of.
+/// \param dev is the PI device that the native allocation will be resident on
 /// \param nativeHandle is the native handle of mem.
-__SYCL_EXPORT pi_result piextMemGetNativeHandle(pi_mem mem,
```

Comment:
Since this changes API we should increment `_PI_H_VERSION_MAJOR`

Reply:
Aha yes good shout thanks.